R/deployApp.R

```diff
@@ -192,6 +192,9 @@ deployApp <- function(appDir = getwd(),
   appDir <- normalizePath(appDir)
 
   # create the full path that we'll deploy (append document if requested)
+  # TODO(HW): we use appPrimaryDoc here, but we have not inferred it yet
+  # so the appPath will different deneding on whether it's explicitly
+  # supplied or inferred from the files in the directory.
```

Comment:
Related: `appPath` is also known as the variable `recordDir`, which is _NOT_ always a directory!

Reply:
FWIW I really dislike the very nested directories that `recordDir()` uses and at some point I'll propose something more compact.